src/main/java/org/commonjava/util/partyline/impl/infinispan/InfinispanJF.java

```diff
@@ -555,7 +555,7 @@ public int read() throws IOException
                 }
 
             }
-            if ( block.hasRemaining() )
+            if ( block.hasRemaining() ) // TODO: seems it should be '!hasRemaining'
```

Comment:
This looks contradictory with the block.isEOF()...it's probably a copy/paste error.